lib/cocoapods/installer/analyzer.rb

```diff
@@ -408,9 +410,9 @@ def generate_targets(resolver_specs_by_target)
           end
         end
         aggregate_targets.each do |target|
-          target.search_paths_aggregate_targets = aggregate_targets.select do |aggregate_target|
+          target.search_paths_aggregate_targets.concat(aggregate_targets.select do |aggregate_target|
             target.target_definition.targets_to_inherit_search_paths.include?(aggregate_target.target_definition)
-          end
+          end).freeze
```

Comment:
huh... surprised that freezing here makes a difference

Reply:
It doesn’t, it just makes sure this array isn’t mutated later, since I now cache things based on it